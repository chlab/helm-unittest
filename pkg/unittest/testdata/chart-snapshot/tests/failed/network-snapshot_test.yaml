suite: test ingress
templates:
  - templates/network.yaml

tests:
  - it: should fail as pattern not found
    asserts:
      - hasDocuments:
          count: 1
      - matchSnapshot:
          path: metadata.labels
          matchRegex:
            pattern: .*not-in-snapshot.*
