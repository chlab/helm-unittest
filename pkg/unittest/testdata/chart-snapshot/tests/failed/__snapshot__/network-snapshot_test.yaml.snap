should fail as pattern not found:
  1: |
    app: test-cluster
    app.kubernetes.io/version: null
